packages/react/src/components/diffViewer/DiffViewer.tsx

```diff
@@ -1,22 +1,55 @@
-import {ComponentProps, FC} from 'react';
-import ReactDiffViewer, {ReactDiffViewerStylesOverride} from 'react-diff-viewer';
+import {FunctionComponent, ReactNode} from 'react';
+import {parseDiff, Decoration, Diff, Hunk} from 'react-diff-view';
 
 import {BlankSlate} from '../blankSlate';
 
-/**
- * @deprecated Use oldValue and newValue Props.
- */
-export interface DiffViewerPropsDeprecated {
-    /**
-     * @deprecated Use oldValue instead.
-     */
-    first: string;
-    /**
-     * @deprecated Use newValue instead.
-     */
-    second: string;
+enum DIFF_VIEWER_VIEW_TYPE {
+    SPLIT = 'split',
+    UNIFIED = 'unified',
+}
+
+interface Hunk {
```

Comment:
Doesn't that cause a problem? We are defining an interface with the same name as an import 🤔

Reply:
No but it's not clean, I'm changing